docs/book/src/user/quick-start.md

```diff
@@ -65,46 +159,144 @@ it detects that there is only an `aws` infrastructure provider and so it uses th
 
 </aside>
 
-
 <aside class="note">
 
 <h1> What topology will be used for my cluster? </h1>
 
-The `clusterctl config cluster` by default uses cluster templates which are provided by the infrastructure providers.
-See the provider's documentation for more information.
+The `clusterctl config cluster` command by default uses cluster templates which are provided by the infrastructure
+providers. See the provider's documentation for more information.
 
-See [`clusterctl config cluster`](../clusterctl/commands/config-cluster.md) for details about how to use alternative sources.
-for cluster templates.
+See the `clusterctl config cluster` [command][clusterctl config cluster] documentation for
+details about how to use alternative sources. for cluster templates.
 
 </aside>
 
 <aside class="note warning">
 
 <h1>Action Required</h1>
 
-If the cluster template defined by the infrastructure provider expects some environment variables, user
+If the cluster template defined by the infrastructure provider expects some environment variables, you
 should ensure those variables are set in advance.
 
-If you have followed [provided specific requirements](#provider-specific-prerequisites), everything should be already set
-at this stage. Otherwise, you can look at [`clusterctl config cluster`](../clusterctl/commands/config-cluster.md) for details about how to discover the list of
-variables required by a cluster templates.
+Instructions are provided for common providers below.
+
+Otherwise, you can look at the `clusterctl config cluster` [command][clusterctl config cluster] documentation for details about how to
+discover the list of variables required by a cluster templates.
 
 </aside>
 
+
+### Required configuration for common providers
+
+Depending on the infrastructure provider you are planning to use, some additional prerequisites should be satisfied
+before configuring a cluster with Cluster API.
+
+{{#tabs name:"tab-installation-infrastructure" tabs:"AWS,Azure,Docker,GCP,vSphere,OpenStack,Metal3"}}
+{{#tab AWS}}
+
+Download the latest binary of `clusterawsadm` from the [AWS provider releases] and make sure to place it in your path.
+
+
+```bash
+$ export AWS_SSH_KEY_NAME=default
```

Comment:
Might be good to show the AWS_REGION env var again, since a user might want to create the cluster in a different region.
```suggestion
$ export AWS_REGION=us-east-1
$ export AWS_SSH_KEY_NAME=default
```